hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/gas/SystemContractGasCalculatorTest.java

```diff
@@ -37,14 +37,13 @@ void setUp() {
 
     @Test
     void returnsMinimumGasCostForViews() {
-        assertEquals(100L, subject.viewGasRequirement());
+        assertEquals(1198L, subject.viewGasRequirement());
```

Comment:
This is expected, correct?  Was the feature flag set in such a way as to return the 100 gas previously?